pages/event-types/index.tsx

```diff
@@ -420,10 +417,10 @@ const CreateNewEventDialog = ({
       <DialogContent>
         <div className="mb-8">
           <h3 className="text-lg font-bold leading-6 text-gray-900" id="modal-title">
-            Add a new {teamId ? "team " : ""}event type
+            {t("add_new_event_type")}
```

Comment:
There is a variation here in the case of creating a team event type. Can you update this, please?
Looks great overall! Thanks for helping out!

Reply:
I have tried to do it, but when it was for teams, the name of the team does not appear, so I decided to remove it and I don't know if it would be better to put a separate title for the team.

Reply:
And there is another problem, in non-English translations the formation of sentences is different, for example in English equipment goes in the middle, but for example in Spanish it goes in the end, so it is better to consider the option of putting another pair of equipment.